custom/icds/location_reassignment/utils.py

```diff
@@ -98,3 +98,6 @@ def split_location_name_and_site_code(name):
         return match.groups()
     return name, None
 
+
+def append_location_name_and_site_code(name, site_code):
+    return f"{name.rstrip()} [{site_code}]"
```

Comment:
Why just `rstrip`?

Reply:
just trying to handle the case where someone left an empty space in the end of the name and avoid cases like "location name    [123]". If they left it in the beginning, that should be okay

Reply:
I could not put multiple spaces in the comment, but i wanted to put something like
location name (extra spaces more than one) [location code]

Reply:
@orangejenny let me know if this is cool, feel free to merge if it is or drop a note, this is g2g.

Reply:
Sounds good to me, merging.